source/catalog.html

```diff
@@ -0,0 +1,279 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page__body">
+
+  <header class="main-header">
+    <a class="main-header__logo">
+      <img src="" width="34" height="38" alt="Кэт Энерджи">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+      <ul class="main-nav__list site-list">
+        <li class="site-list__item ">
+          <a href="">
+            Главная
+          </a>
+        </li>
+        <li class="site-list__item site-list__item--active">
+          <a href="">
+            Каталог продукции
+          </a>
+        </li>
+        <li class="site-list__item">
+          <a href="">
+            Подбор программы
+          </a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+  <main class="main-container">
+    <h1 class="main-container-title">
+      Каталог продукции
+    </h1>
+
+    <section class="catalog">
+      <div class="product">
+        <h1 class="visually-hidden">
+          Каталог продукции Кэт Энерджи
+        </h1>
+        <ul class="product__list">
+          <li class="product-card">
+            <a class="product-card__image-link" href="#">
+              <img class="product-card__image" src="" width="68" height="86"
+                alt="Питание Кэт Энерджи Про со вкусом курицы 500 грамм">
+            </a>
+            <a class="product-card__title-link" href="#">
+              <h3 class="product-card__title">Cat Energy PRO 500 г</h3>
+            </a>
+            <dl class="product-card__specifications">
+              <dt class="product-card__specifications-definition">Масса</dt>
+              <dd class="product-card__specifications-meaning">500 г</dd>
+              <dt class="product-card__specifications-definition">Вкус</dt>
+              <dd class="product-card__specifications-meaning">Курица</dd>
+              <dt class="product-card__specifications-definition">Цена</dt>
+              <dd class="product-card__specifications-meaning">700 Р.</dd>
+            </dl>
```

Comment:
Довольно странно называть `specifications-definition` слово "Масса". Потому что, для начала, "definition" это `<dd>`, он так и сокращен от "description definition".

Потом, ты не делаешь это в нейминге, но обрати внимание, что у тебя `specifications` работает почти как отдельный БЭМ-блок. Он имеет общее название на обертке, а дальше через `-` имеет элементы. Если такое вылезает в коде, то просто делай отдельный блок, не надо эту имитацию использовать.

```suggestion
            <dl class="product-card__specifications specifications">
              <dt class="specifications__definition">Масса</dt>
              <dd class="specifications__meaning">500 г</dd>
              <dt class="specifications__definition">Вкус</dt>
              <dd class="specifications__meaning">Курица</dd>
              <dt class="specifications__definition">Цена</dt>
              <dd class="specifications__meaning">700 Р.</dd>
            </dl>
```

Reply:
Поняла, исправила